lib/payment.js

```diff
@@ -57,25 +58,36 @@ class Payment {
      * @param {Address} sender - Senders address
      * @param {Address} recipient - Recipient address
      * @param {Wallet|NahmiiProvider} [walletOrProvider] - An optional Wallet or NahmiiProvider instance
+     * @param {String} [senderPayload] - Optional stringified payment sender data payload
      * @param {String} [senderRef] - Optional uuid identifying the payment. Must be unique per sender wallet. Random if undefined.
      *
      * @example
-     * // Normal, random sender reference
+     * // Normal, no sender payload and random sender reference
      * const senderWallet = new nahmii.Wallet(...);
      * const recipientWallet = new nahmii.Wallet(...);
      * const paymentAmount = nahmii.MonetaryAmount.from(...);
      * const payment = new nahmii.Payment(paymentAmount, senderWallet.address, recipientWallet.address, senderWallet);
      *
-     * // Advanced, semantic sender reference. See: https://github.com/uuidjs/uuid
+     * // Advanced, sender payload and random sender reference
      * const uuidNamespace = '706ac453-2691-41af-9fde-ac5f787da1ec';
```

Comment:
uuidNamespace probably not needed in this example.

Reply:
True. Correcting.